sql/amo_prod/desktop_addons_by_client_v2/query.sql

```diff
@@ -0,0 +1,41 @@
+WITH per_client AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_id,
+    sample_id,
+    ARRAY_CONCAT_AGG(
+      ARRAY(
+        SELECT AS STRUCT
+          REPLACE(REGEXP_EXTRACT(addon, "(.+):"), '%40', '@') AS id,
```

Comment:
I think we should also replace `%7D` with `}` and `%7B` with `{`.

Reply:
I've extracted this to a Javascript UDF where we can do proper URI decoding. It makes the query take a bit longer to run, but gets us significantly more correct results. That boosted us to 99% coverage of the v1 data.